### ISSUE TITLE
Modal examples needed for testing

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-react#1044]
As noted in https://github.com/patternfly/patternfly-next/issues/925, some working examples of the modal are needed to better understand how the aria and role attributes might affect the behavior of a modal in screen readers. 

Can we have a WIP PR created with the following examples? 

- example with `aria-label` (refer to [core](https://pf-next.com/components/ModalBox/examples/) example "ModalBox No Header Example" and use of `aria-label` on `.pf-c-modal-box`)
- example with `aria-labelledby`
- example with `aria-describedby`
- example with contents included in the modal other than the `aria-describedby` contents, header, and buttons (refer to [core](https://pf-next.com/components/ModalBox/examples/) example "ModalBox No Header Example" and use of `<span id="modal-no-header-description">`)
- one example where contents are text and one where contents are a form would be ideal

For each example above, there should be two examples: 
- one with `role="dialog" aria-modal="true"` 
- and one without these attributes

**Comment by rachael-phillips** [comment]
Closing this issue, completed
